src/components/map-components/testing-location-list-item.js

```diff
@@ -19,6 +20,14 @@ export default function TestingLocationListItem(props) {
     onTestingLocationExpand(id, itemIndex, title, isExpanded);
   };
 
+  const checkDate = (date) => {
```

Comment:
as a convention, I like to name things that return booleans prefixed with "is" "should", "can" etc

so this one would be `isNewLocation()`